packages/base/imports.js

```diff
@@ -0,0 +1,294 @@
+module.exports = {
```

Comment:
Would it make sense to move import sorting rule to this package? (currently in es6)

Reply:
No, it doesn't come from `eslint-plugin-import` and it's switched `"off"`. Although I think I'm going to switch it on, now that you mention it.